api/decorators/Validators.ts

```diff
@@ -47,6 +47,23 @@ export function IsValidName(validationOptions?: ValidationOptions) {
   return templatedValidationDecorator(NameValidator, validationOptions);
 }
 
+@ValidatorConstraint()
+class HandleValidator implements ValidatorConstraintInterface {
+  regex = /^[a-zA-Z0-9-]+$/;
```

Comment:
nit: let's add a small comment here on what characters get matched with this regex (just so that developers who might not know how to parse regex still understand what's happening here).